src/core/css/SearchResult.scss

```diff
@@ -48,19 +48,13 @@
   }
 }
 
-.SearchResult-author,
-.SearchResult-users {
+.SearchResult-author-users {
   color: $sub-text-color;
-  display: inline-block;
+  display: block;
   font-size: $font-size-s;
   font-weight: normal;
-  margin: 0;
+  margin-right: 0.5rem;
```

Comment:
@tofumatt since it uses margin-right, should I use margin-end from the mixin file instead?